src/Schema/Events/Click.ts

```diff
@@ -1627,6 +1627,7 @@ export interface ClickedPublish {
   context_module: ContextModule
   artwork_id: string
   label: string
+  shipping_preset_id?: string
 }
```

Comment:
these ones I want to look into more - maybe its ok to have such a specific optional field on a very generic artwork click event (?)

Reply:
Yeah, that's the idea. There are already specific actions (`ClickedPublish`, `ClickedSave`). They only get triggered in CMS when hitting the CTAs, so by adding this we can always capture the shipping preset id on saving and publishing.